sql/portal-data-common.sql

```diff
@@ -669,6 +669,14 @@ insert into Region (regionId, countryId, regionCode, name, active_) values (2020
 insert into Region (regionId, countryId, regionCode, name, active_) values (202025, 202, 'ZG', 'Zug', TRUE);
 insert into Region (regionId, countryId, regionCode, name, active_) values (202026, 202, 'ZH', 'Zürich', TRUE);
 
+##
+## List types for account entries
+##
+
+insert into ListType (listTypeId, name, type_) values (14000, 'billing', 'com.liferay.account.model.AccountEntry.address');
+insert into ListType (listTypeId, name, type_) values (14001, 'billing-and-shipping', 'com.liferay.account.model.AccountEntry.address');
+insert into ListType (listTypeId, name, type_) values (14002, 'shipping', 'com.liferay.account.model.AccountEntry.address');
+
```

Comment:
I think in our case it's probably better to use `ListTypeLocalService#addListType` in a `PortalLifecycleListener` since this is only relevant when the `accounts` modules are deployed.